tests/flow/test_pending_queries_limit.py

```diff
@@ -42,27 +67,15 @@ def test_01_query_limit_config(self):
         self.env.assertEquals(max_queued_queries, 10)
 
     def stress_server(self):
+        global error_encountered
+
         threads = []
-        connections = []
         threadpool_size = self.conn.execute_command("GRAPH.CONFIG", "GET", "THREAD_COUNT")[1]
         thread_count = threadpool_size * 5
 
-        # init connections
-        for i in range(thread_count):
-            connections.append(self.env.getConnection())
-
-        # invoke queries
-        for i in range(thread_count):
-            con = connections.pop()
-            t = threading.Thread(target=issue_query, args=(con, SLOW_QUERY))
-            t.setDaemon(True)
-            threads.append(t)
-            t.start()
-
-        # wait for threads to return
-        for i in range(thread_count):
-            t = threads[i]
-            t.join()
+        results = run_test_multiproc(self.env, thread_count, issue_query, (SLOW_QUERY,))
```

Comment:
do we need the extra `,` in `(SLOW_QUERY,)` ?

Reply:
To create a tuple with only one item, you have add a comma after the item, unless Python will not recognize the variable as a tuple.

taken from here: https://www.w3schools.com/python/gloss_python_tuple_one_item.asp